evalml/automl/engine/dask_engine.py

```diff
@@ -0,0 +1,83 @@
+import joblib
+from dask.distributed import Client
+
+from evalml.automl.engine.engine_base import (
+    EngineBase,
+    EngineComputation,
+    evaluate_pipeline,
+    score_pipeline,
+    train_pipeline
+)
+
+
+class DaskComputation(EngineComputation):
+    """A Future-like wrapper around jobs created by the DaskEngine."""
+
+    def __init__(self, dask_future):
+        self.work = dask_future
+        self.meta_data = {}
+
+    def done(self) -> bool:
+        """Is the computation done?"""
+        return self.work.done()
+
+    def get_result(self):
+        """Get the computation result.
+        Will block until the computation is finished.
+
+        Raises Exception: If computation fails. Returns traceback.
+        """
+        return self.work.result()
```

Comment:
In dask, when you call `result` it returns the result of the function you sent to the worker. So the value of our `get_result` depends on the job. For pipeline evaluation it's the scores, pipeline class, and log messages. For pipeline scoring it's the scores and for pipeline training it is just the trained pipeline. 

Since the computation itself is could be anything I think the docsting is adequate ("Get the computation result.").

Reply:
Ahhh got it got it. that makes sense, thanks!